microshift_cli_ref/microshift-cli-overview.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="cli-tools-overview"]
+= {product-title} CLI tools
+include::_attributes/attributes-microshift.adoc[]
+:context: cli-tools-overview
+
+toc::[]
+
+A user builds, deploys, and manages applications, as well as managing clusters while working with {product-title}.
+
+{product-title} can use different command-line interface (CLI) tools that simplify these tasks by enabling users to perform various administration and development operations from the terminal.
+These tools expose simple commands to manage the deployments, as well as interact with each component of the system.
+
+In addition to built-in `microshift` command types and Linux CLI tools, the optional OpenShift CLI (`oc`) tool is available for you to use if you are already familiar with {OCP} and Kubernetes.
+
+The enabled subset of commands available with the `oc` tool are commonly used by {product-title} users. It helps both cluster administrators and developers to perform operations across {product-title} using the terminal.
+
+[role="_additional-resources"]
```

Comment:
Per the [contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies), this should be formatted as:
````
[role="_additional-resources"]
[id="additional-resources_cli-tools-overview"]
== Additional resources
````